pkg/apis/pingcap/v1alpha1/types.go

```diff
@@ -1997,6 +2048,8 @@ type RestoreSpec struct {
 	To *TiDBAccessConfig `json:"to,omitempty"`
 	// Type is the backup type for tidb cluster.
 	Type BackupType `json:"backupType,omitempty"`
+	// RestoreType is the restore type.
+	RestoreType RestoreType `json:"restoreType,omitempty"`
```

Comment:
Some comments to explain the default value? or can we reuse the `BackupType?

Reply:
default value will be `full`.
maybe we need use `RestoreType`, `PiTR` restore need a snapshot backup and a log backup, if we use `BackupType`, it is hard to indicate both of them.